inc/class_core.php

```diff
@@ -519,6 +521,8 @@ public function get_asset_url($path = '', $use_cdn = true)
 			$url = $path;
 		}
 
+		$plugins->run_hooks('get_asset_url_end', $url);
```

Comment:
But this is quite an addition IMO and it should be checked.

@mybb/developers @Destroy666x @JN-Jones 

This method runs from 6/7 to 9 nine times a page in a fresh installation.